### ISSUE TITLE
Stutter when Screen Mirroring is active

### ISSUE THREAD

**Issue by melgu** [winleafs/Winleafs#121]
**Describe the bug**
There is noticeable stuttering in the OS when doing anything (like moving a window).
There's a short stutter whenever the lights gets updated. Meaning: refresh rate of 1 = 1 stutter per second, refresh rate of 10 = 10 stutters per second.

Aurora: 27 Panels.
PC: 2x GTX 1080, Intel i7 6900K, 4K screen resolution at 60 Hz.

**To Reproduce**
1. Enable Screen Mirroring

**Logs**
[log.log](https://github.com/winleafs/Winleafs/files/4577857/log.log)

**Comment by StijnOostdam** [comment]
Hey, thanks for the bug report. Your specs are definitely good enough to run screen mirror without lag with at least a refresh rate of 5. I will do some research on how the stutter can still occur.

**Comment by StijnOostdam** [comment]
Follow up question: is there also stuttering when playing a game or watching a video?

**Comment by melgu** [comment]
Yes. The stutter happens globally. It affects the entire system.

**Comment by melgu** [comment]
Still stutters with an Intel 13900K and an RTX 4090.
Having HDR enabled makes the stuttering much worse, but even without HDR it stutters noticeably.
